server/controllers/packFeed.js

```diff
@@ -0,0 +1,90 @@
+const db = require('../db/database');
+var getUserPacksId = (userId, res) => {
+  // db.connect();
+  //replace below query with: `select * from public."packUsers" where public."packUsers"."userId" = ${userId} or public."packUsers"."userId" = 15`
+  //to test if it works with multiple groups being returned by query.
+  db.query(`select * from pack_users where pack_users.user_id = ${userId} `).then((partOf) => {
+    var promises = [];
+    for (var i = 0; i < partOf.rows.length; i++) {
+      promises.push(
+        db.query(`select * from packs where packs.pack_id = ${partOf.rows[i].pack_id}`)
+      );
+    }
+    var toSendBack = [];
+    Promise.all(promises).then((result) => {
```

Comment:
This this query will also work with just: 
SELECT pack_id FROM pack_users WHERE user_id = ${user you want to search}
unless I'm misunderstanding

Reply:
I think i did it this way because multiple results can be returned from a query, and i need to run a set of queries for each result so i package them into an array of promises and return the result from that